README.md

```diff
@@ -266,6 +266,72 @@ Simplified tree diagram
       }
   ```
 
+### Get all threads for a user
+  GET /messaging/me/
```

Comment:
I'd also expect that `GET /messaging` would return all the threads.  can you add this to the urls and docs?

Reply:
You mean all the threads for the requesting user or all the threads total?

Reply:
requesting user. same. thing as GET /messaging/me